pagerduty/table_pagerduty_service_integration.go

```diff
@@ -0,0 +1,175 @@
+package pagerduty
+
+import (
+	"context"
+
+	"github.com/PagerDuty/go-pagerduty"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tablePagerDutyServiceIntegration(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "pagerduty_service_integration",
+		Description: "A service integration is an integration that belongs to a Pagerduty service.",
+		List: &plugin.ListConfig{
+			Hydrate: listPagerDutyServiceIntegrations,
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:    "service_id",
+					Require: plugin.Required,
```

Comment:
How many services does a PagerDuty account typically have? Would there be too many results if we used a parent hydrate here instead?

Reply:
@cbruno10 There are no limitation on creating the service or integration per service
Parent hydrate can reduce the query speed, if there are too many integration present per service.